google/cloud/internal/async_read_write_stream_impl_test.cc

```diff
@@ -32,7 +32,7 @@ namespace {
 using ::google::cloud::testing_util::IsOk;
 using ::google::cloud::testing_util::MockCompletionQueueImpl;
 using ::testing::_;
-using ::testing::ReturnRef;
+using ::testing::Return;
```

Comment:
These remaining tests may be able to use `Return(nullptr)`, but I am less certain.